### DISCUSSION TITLE
Lazer crash on finishing last map in queue.

### DISCUSSION THREAD

**Discussion by TheFantasticLoki** [ppy/osu#17149] [Q&A] [answered]
Was playing in multiplayer with a random on TJ Davis - Super Sonic Racing (Woey) [Come on now, race!] and when the map finished I immediately pressed esc to skip score screen and then the game froze at the lobby screen then crashed. It was the last map in the queue.
I was host but not the original host of the lobby. (just adding details)
Here are my log files:
[legacy-ipc.log](https://github.com/ppy/osu/files/8199774/legacy-ipc.log)
[network.log](https://github.com/ppy/osu/files/8199775/network.log)
[performance.log](https://github.com/ppy/osu/files/8199776/performance.log)
[runtime.log](https://github.com/ppy/osu/files/8199777/runtime.log)
[updater.log](https://github.com/ppy/osu/files/8199778/updater.log)
[database.log](https://github.com/ppy/osu/files/8199779/database.log)

**Comment by bdach** [comment, marked as answer]
the logs seem like they've cut out entirely at the time of crash and don't show any discernible errors. please check whether there are any errors that look related in windows event viewer.

**Reply by TheFantasticLoki** [reply]
2 errors:
`Faulting application name: osu!.exe, version: 0.0.0.0, time stamp: 0x61e1d8d9
Faulting module name: coreclr.dll, version: 6.0.222.6406, time stamp: 0x61e1d09e
Exception code: 0xc0000005
Fault offset: 0x00000000001cd72f
Faulting process id: 0x4958
Faulting application start time: 0x01d83222bac97bab
Faulting application path: C:\Users\astey\AppData\Local\osulazer\app-2022.306.0\osu!.exe
Faulting module path: C:\Users\astey\AppData\Local\osulazer\app-2022.306.0\coreclr.dll
Report Id: 6befbd86-038b-4a01-8153-5628ac3fb198
Faulting package full name: 
Faulting package-relative application ID: `

`Application: osu!.exe
CoreCLR Version: 6.0.222.6406
.NET Version: 6.0.2
Description: The process was terminated due to an internal error in the .NET Runtime at IP 00007FFECD90D72F (00007FFECD740000) with exit code c0000005.
`

**Reply by bdach** [reply]
unfortunately those look driver-related and too vague to know what is going on for sure, sorry. please try updating your graphics drivers and/or disabling shadowplay if you have it on.

**Reply by TheFantasticLoki** [reply]
I'll be marking this as answered as it seems not much more can be gathered on the subject and in all likely hood it is just a driver issue since I have hundreds installed.
